src/main/java/com/esri/samples/na/offline_routing/OfflineRoutingSample.java

```diff
@@ -97,6 +99,15 @@ public void start(Stage stage) throws Exception {
       routeTask = new RouteTask("./samples-data/san_diego/sandiego.geodatabase", "Streets_ND");
       routeTask.loadAsync();
 
+      // add a graphics overlay to show the boundary
+      Envelope envelope = new Envelope(new Point(-13045352.223196, 3864910.900750, 0, SpatialReferences
```

Comment:
@tschie Where did you get these points from? Just wondering because if I was a user I would like to know how to obtain these points for when I create an application using this. I tried out some different methods on RouteTask but didn't come up with a successful ways to get the extent of the geodatabase being used.